lib/SILGen/SILGenType.cpp

```diff
@@ -1087,6 +1088,14 @@ class SILGenType : public TypeMemberVisitor<SILGenType> {
     for (Decl *member : theType->getABIMembers())
       visit(member);
 
+    // Types can have stored properties that aren't included in
+    // `getABIMembers()`, so we have to handle those manually
+    for (VarDecl *VD : theType->getStoredProperties()) {
+      if (!visitedVarDecls.count(VD)) {
```

Comment:
Instead of using a set here, you can just skip stored properties in the first loop